projects/managed-hubs/support.md

```diff
@@ -1,118 +1,214 @@
+5. **If an Incident**, go to [](support:incident-response).
+6. **If not an Incident**, go to [](support:non-incident-response).
+
+(support:non-incident-response)=
+### Non-incident response process
 
-- When you receive a ticket, communicate that you have received it and note the next steps you’ll follow to resolve it (even if this just means investigating because you're not sure what is wrong).
-- If needed, open a [support issue and add it to the backlog](support:issues).
-  Note that we should still use Freshdesk for support communication on this request, the GitHub issue is just for 2i2c team members to coordinate.
-- Identify an engineering team member that can pick up this work.
-- Give the user updates every day or two as we work to resolve the issue.
-- When it is resolved, tell the user what you’ve done to resolve the issue.
-- Close any FreshDesk tickets and support issues that are related to this request.
-- In some cases, it might be useful to also write down [a private note in the FreshDesk ticket](https://support.freshdesk.com/support/solutions/articles/37580-private-notes-for-internal-sharing)
-  with a short summary of the situation and/or add links to relevant discussions
-  from other platforms.
+1. **Respond within 24 working hours**. Acknowledge receipt of the support request and let the {term}`Community Representative` know about any investigation we have done thus far. 
+2. **Spend 30 minutes trying to resolve**. If you believe you can resolve the issue within 30 minutes, try resolving it yourself.
+   1. If you resolve the issue, then jump to the "Communicate resolution" step.
+   2. If you don't believe you can resolve the issue in 30 minutes, jump to the next step.
+3. **Open an engineering issue**. If this is a {term}`Change Request` or {term}`Guidance Request` and you cannot resolve the issue within 30 minutes, then open a support issue for the team to discuss.
+   [{bdg-primary}`Open a general issue`](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=&template=01_new-issue.yml) that describes the request and any next steps we should take. Add the {guilabel}`support` label to it.
+4. **Communicate status**. Once we have an issue created to track the next steps, send a message to the Community Representative letting them know about the situation and what the next steps will be. Here's a template to help guide you:
 
-## How to prioritize support requests
+   ```
+   Hello { NAME }, thanks for sending us your request. We've concluded that
+   this is not related to an outage or diminished service, so we'll incorporate
+   this request into our team's work planning process. We'll send an update
+   when we've got a plan for completing this request.
+   ```
+5. **Prioritize the request**. Work with our team to decide how we should prioritize this request relative to the other work we need to do. Follow the [how to prioritize Change and Guidance Requests guide](support:prioritize-requests).
```

Comment:
Maybe also talk with the PMs to agree on prioritization levels.

Reply:
yeah I agree - this is a process that the Project/Product managers should be involved with.